app/controllers/spree/admin/adjustments_controller.rb

```diff
@@ -18,7 +18,12 @@ def update_order
       end
 
       def collection
-        parent.all_adjustments.eligible
+        order_adjustments = parent.adjustments.where.not(originator_type: 'EnterpriseFee')
+        admin_adjustments = parent.adjustments.admin
+        payment_fees = parent.all_adjustments.payment_fee.eligible
+        shipping_fees = parent.all_adjustments.shipping
+
+        order_adjustments.or(admin_adjustments) | payment_fees.or(shipping_fees)
```

Comment:
does the `|` become an AR merge?

Reply:
Yeah they get combined :+1:

Reply:
This does add quite a bit of complexity... are there other types of fees (or other types that could be added in the future) that we'd then have to come and update this query with?

Reply:
Right, or the other way around; are we skipping a particular type of adjustment? How come we can't simply do `parent.all_adjustments`?